charts/alfresco-connector-ms365/Chart.yaml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: v2
+name: alfresco-connector-ms365
+description: A Helm chart for deploying Alfresco connector ms365 service
+type: application
+version: 0.1.0-alpha.1
```

```suggestion
version: 0.1.0-alpha.0
```

we always start to count from zero